web/.gitignore

```diff
@@ -0,0 +1,3 @@
+/INSTALL.txt
+/README.txt
+/example.gitignore
```

Comment:
This line is being flagged because there's no newline at the end of the file.

Reply:
I went ahead and added that line ending just to keep it clean. That change was autogenerated by the uninstall script.